velox/dwio/dwrf/test/ColumnStatisticsBase.h

```diff
@@ -0,0 +1,966 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <gtest/gtest.h>
+#include "velox/dwio/common/Statistics.h"
+#include "velox/dwio/dwrf/writer/StatisticsBuilder.h"
+
+namespace facebook::velox::dwrf {
+class ColumnStatisticsBase {
```

Comment:
ColumnStatisticsBase has virtual functions, so it needs to have a virtual destructor.

(I'll take a look at updating our build flags to fail on this case)

Reply:
I moved the implementation of `options()` and `context()` to `ColumnStatisticsBase.h` so that there is no need to add a virtual destructor.